### PR TITLE
OPCBUGS#29750: Fixup the YAML in the example CRD for OCP 4.15

### PR DESCRIPTION
**Pull request by michaelryanpeter**
Version(s): 4.15

Issue: [OCPBUGS-29750](https://issues.redhat.com/browse/OCPBUGS-29750)

Link to docs preview:
* [Creating a CRD > Example YAML file](https://71969--ocpdocs-pr.netlify.app/openshift-enterprise/latest/operators/understanding/crds/crd-extending-api-with-crds#crd-creating-custom-resources-definition_crd-extending-api-with-crds)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by adellape — APPROVED** [review]

**Comment by michaelryanpeter** [comment]
@Xia-Zhao-rh or @jianzhangbjz: PTAL.

File: modules/crd-creating-crds.adoc [hunk 1]
```diff
@@ -28,14 +28,29 @@ metadata:
...
+                replicas:
+                  type: integer
   scope: Namespaced <5>
   names:
     plural: crontabs <6>
     singular: crontab <7>
     kind: CronTab <8>
     shortNames:
-    - ct <9>
+      - ct <9>
```
Comment (Xia-Zhao-rh) [inline modules/crd-creating-crds.adoc:53]: no blank here
```
shortNames:
 - ct <9>
 ```

File: modules/crd-creating-crds.adoc [hunk 2]
```diff
@@ -28,14 +28,29 @@ metadata:
...
+                replicas:
+                  type: integer
   scope: Namespaced <5>
   names:
     plural: crontabs <6>
     singular: crontab <7>
     kind: CronTab <8>
     shortNames:
-    - ct <9>
+     - ct <9>
```
Comment (Xia-Zhao-rh) [inline modules/crd-creating-crds.adoc:53]: still have one whitespace, please delete it.

Comment (michaelryanpeter) [reply modules/crd-creating-crds.adoc]: Fixed https://github.com/openshift/openshift-docs/pull/71969/files#diff-7ddc0a682fe2c5e55c86cd502c63bb313c7509f8dbb35b318a5ffd3a96a9ad8fL37-L38

**Comment by Xia-Zhao-rh** [comment]
LGTM

**Pull request merged**